more/free-programming-interactive-tutorials-en.md

```diff
@@ -133,6 +141,12 @@
 * [Top 10 Data Science Projects for Beginners - KDnuggets](https://www.kdnuggets.com/2021/06/top-10-data-science-projects-beginners.html)
 
 
+### DevOps
```

Comment:
both of these fit better in playgrounds.

Reply:
Thanks for the review  and help i have moved them to the playground section